cloudsync_onedrive.py

```diff
@@ -37,8 +37,31 @@
 
 import quickxorhash
 
+# TODO: import this
+class Namespace(NamedTuple):
+    name: str
+    id: Optional[str]
+    is_parent: bool = False
 
-__version__ = "0.1.20" # pragma: no cover
+class Drive(NamedTuple):
```

Comment:
if you use dataclass instead of namedtuple, Drive and Site can inherit from Namespace and just change the __init__ to default is_parent	.  Might be nicer?

Reply:
yeah, was planning to come back to this once I have a cloudsync version that I can import Namespace from. Drive and Namespace are essentially the same, so may not even need Drive. Site can probably derive from Namespace.